Code-the-Dream-School/bb-practicum-team-3-front
# Error component

## Description
Add error component and refactor search results page to include error component. 
First, the page displays the loading component, if there is an error then displays the error component, else displays search results. Before it would just display the loading component forever.


## Related Issue



## Acceptance Criteria



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before
![image](https://user-images.githubusercontent.com/74852114/235836828-22a66f2d-f7a7-41ec-9d3a-bf2bbf199a82.png)




### After
![image](https://user-images.githubusercontent.com/74852114/235836766-5498b502-cc60-4162-a8e1-0add89865716.png)



## Testing Steps / QA Criteria
Run both front and backend servers and click on one of the popular destination links. 
Should first display loading component then the search results. 
To test error shut down the backend server. Should first display loading component then the error component.
